docs/book/collaborate/zenml-store.md

```diff
@@ -1,240 +0,0 @@
----
-description: How to manage stacks in a centralized location with ZenStores
```

Comment:
Since ZenStore is still a concept, so I don't think this should be totally removed.